components/docs-chef-io/content/automate/ha_cert_rotation.md

```diff
@@ -36,28 +36,30 @@ The certificate rotation is also required when the key for a node, client, or CA
 {{< /note >}}
 
 ### Rotate Cluster Certificates
+
 If you want to rotate certificates of the entire cluster using single command, then you can follow the below commands:
 
 To rotate certificates of entire cluster using single command, we need a certificate template.
+
 - To generate certificate template use below command
-```
-  chef-automate cert-rotate generate-certificate-config certificate-config.toml
-```
 
-now we can find our certificate template in `certificate-config.toml` file, please edit the file and put required certificate paths
+    ```sh
+        chef-automate cert-rotate generate-certificate-config certificate-config.toml
+    ```
 
-- To rotate the certificates use below command
+    Now, we can find our certificate template in `certificate-config.toml` file, please edit the file and put required certificate paths
```

Comment:
Don't use _we_, _our_, or _please_. We aren't doing anything, the customer is doing something. Stick to the imperative, for example ("Find the certificate template and add the required certificate paths.")